### PR TITLE
Entra Rebrand

### PR DESCRIPTION
**Pull request by DidunAyodeji**
Updating all Azure AD terms to the Microsoft Entra ID terms in all md files.

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
The updated changes to Microsoft Entra ID terminology make sense.

## Other Information
Here's a link to the Rebrand guidelines. Note that the script doesn't have all the entries in the glossary that mine has. Let me know if you'd like to see my version of the script. https://review.learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad?branch=main

### CODE REVIEW

File: 1-Calling-MSGraph/1-1-AzureAD/AppCreationScripts/AppCreationScripts.md [hunk 1]
```diff
@@ -123,8 +123,8 @@ Of course, in real life, you might already get the password as a `SecureString`.
 #### Option 3 (Interactive, but create apps in a specified tenant)
 
   if you want to create the apps in a particular tenant, you can use the following option:
-- open the [Azure portal](https://portal.azure.com)
-- Select the Azure Active directory you are interested in (in the combo-box below your name on the top right of the browser window)
+- open the [Microsoft Entra admin center](https://entra.microsoft.com)
+- Select the Microsoft Entra ID you are interested in (in the combo-box below your name on the top right of the browser window)
```
Comment (jmprieur) [inline 1-Calling-MSGraph/1-1-AzureAD/AppCreationScripts/AppCreationScripts.md]: I think this should be 'select the tenant'

File: 1-Calling-MSGraph/1-3-NationalClouds/README.md [hunk 2]
```diff
@@ -85,12 +85,12 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information, see MSAL.NET's conceptual documentation:
 
 - [MSAL.NET's conceptual documentation](https://aka.ms/msal-net)
-- [Microsoft identity platform (Azure Active Directory for developers)](https://docs.microsoft.com/azure/active-directory/develop/)
+- [Microsoft identity platform (Microsoft Entra ID for developers)](https://docs.microsoft.com/azure/active-directory/develop/)
```
Comment (jmprieur) [inline 1-Calling-MSGraph/1-3-NationalClouds/README.md]: No. This one shouldn't change

File: 2-TokenCache/AppCreationScripts/AppCreationScripts.md [hunk 3]
```diff
@@ -123,8 +123,8 @@ Of course, in real life, you might already get the password as a `SecureString`.
 #### Option 3 (Interactive, but create apps in a specified tenant)
 
   if you want to create the apps in a particular tenant, you can use the following option:
-- open the [Azure portal](https://portal.azure.com)
-- Select the Azure Active directory you are interested in (in the combo-box below your name on the top right of the browser window)
+- open the [Microsoft Entra admin center](https://entra.microsoft.com)
+- Select the Microsoft Entra ID you are interested in (in the combo-box below your name on the top right of the browser window)
```
Comment (jmprieur) [inline 2-TokenCache/AppCreationScripts/AppCreationScripts.md]:
```suggestion
- Select the Microsoft Entra ID tenant you are interested in (in the combo-box below your name on the top right of the browser window)
```

**Review by jmprieur — APPROVED** [review]
Lgtm.  Thanks @DidunAyodeji 
I left a couple of suggestions

File: 1-Calling-MSGraph/1-1-AzureAD/AppCreationScripts/AppCreationScripts.md (hunk 1 above)
Comment (DidunAyodeji) [reply 1-Calling-MSGraph/1-1-AzureAD/AppCreationScripts/AppCreationScripts.md]: Great catch. "Tenant" should've been at the end "Azure Active directory" and the script would've caught it. Will fix right now.

**Pull request merged**